api-definitions/db_positions/api.yaml

```diff
@@ -0,0 +1,296 @@
+swagger: "2.0"
+info:
+  version: 1.0.0
+  title: Debtor Positions API
+  contact:
+    name: PagoPA
+    url: https://www.pagopa.gov.it/
+  x-logo:
+    url: https://www.pagopa.gov.it/assets/images/pagopa-logo.png
+  description: Specification for Debtor Positions API
+host: api.io.italia.it.TOBEDEFINE
+basePath: "/api/v1"
+schemes:
+  - https
+security:
+  - ApiKey: []
+paths:
+  "/payment-notices/{organizationfiscalcode}/payment-notices/{paymentnoticenumber}":
+    post:
+      operationId: createPosition
+      summary: |
+        The PA creates a Debt Position
```

Comment:
I suggest to change every reference to 'PA' to 'Organization' (which is the name we use in IO API)